src/box/box.h

```diff
@@ -626,7 +626,7 @@ API_EXPORT int
 box_session_push(const char *data, const char *data_end);
 
 /**
- * \return current session's unique monotonic identifier (\sa box.session.id)
+ * \return current session's unique monotonic identifier \sa box.session.id
  */
```

Comment:
It looks so, when rendered into HTML:

![image](https://user-images.githubusercontent.com/92749/227646955-3727494d-3507-4f62-9bdf-240d81332ae8.png)

So, maybe, move `\sa <...>` into its own line in the comment too?

Reply:
placed to a separate line:

```diff
--- a/src/box/box.h
+++ b/src/box/box.h
@@ -626,7 +626,8 @@ API_EXPORT int
 box_session_push(const char *data, const char *data_end);
 
 /**
- * \return current session's unique monotonic identifier \sa box.session.id
+ * \return current session's unique monotonic identifier
+ * \sa box.session.id
  */
 API_EXPORT uint64_t
 box_session_id(void);
```